tests/unit/meta/test_meta.py

```diff
@@ -0,0 +1,115 @@
+    # also check skip_validate
+    run(runner, ["--debug", "push", "--dry-run", "--skip_validate"], expected_exit_code=0)
+
+    # should pass now
+    config.gen3.project_id = config.gen3.project_id.replace("XXXX", "")
+    with open('.g3t/config.yaml', 'w') as f:
+        yaml.dump(config.model_dump(), f)
+    run(runner, ["commit", "-m",  "restore-project_id", '.g3t/config.yaml'])
+
+    run(runner, ["--debug", "meta", "validate"], expected_exit_code=0)
+    run(runner, ["--debug", "push", "--dry-run"], expected_exit_code=0)
+
+
+def test_assert_add_specimen_after_init(runner: CliRunner, project_id, tmp_path: pathlib.Path) -> None:
+    """Test meta skeleton handles re-add of data with new specimen"""
+    # change to the temporary directory
+    os.chdir(tmp_path)
+    print(pathlib.Path.cwd())
+    print(project_id)
+
+    run(runner, ["--debug", "--profile", "local", "init", project_id, "--no-server"],
+        expected_files=[".g3t", ".git"])
+
+    # create test files
+    cmds = """
+    mkdir my-project-data
+    mkdir my-read-only-data
+    echo "hello" > my-project-data/hello.txt
+    echo "big-data" > my-read-only-data/big-file.txt
+    ln -s $PWD/my-read-only-data/big-file.txt my-project-data/big-file.txt
+    """.split('\n')
+    for cmd in cmds:
+        run_command(cmd, no_capture=True)
+
+    assert pathlib.Path("my-project-data/hello.txt").exists(), "hello.txt does not exist."
+    assert pathlib.Path("my-read-only-data/big-file.txt").exists(), "my-read-only-data/big-file.txt does not exist."
+    assert pathlib.Path("my-project-data/big-file.txt").exists(), "my-project-data/big-file.txt does not exist."
+
+    files = ["my-project-data/hello.txt", "my-project-data/big-file.txt"]
+    patients = ["P1", "P2"]
+    for f, p in zip(files, patients):
+        run(runner, ["--debug", "add", str(f), "--patient", p], expected_files=[f"MANIFEST/{f}.dvc"])
+
+    run(runner, ["--debug", "meta", "init"], expected_files=["META/DocumentReference.ndjson", "META/Patient.ndjson", "META/ResearchStudy.ndjson", "META/ResearchSubject.ndjson"])
+    run(runner, ["--debug", "meta", "validate"])
+    run(runner, ["commit", "-m",  "init", "MANIFEST/", "META/", ".g3t", ".gitignore"])
+
+    # now add a new specimen
+    files = ["my-project-data/hello.txt", "my-project-data/big-file.txt"]
+    patients = ["P1", "P2"]
+    specimens = ["S1", "S2"]
+    for f, p, s in zip(files, patients, specimens):
+        run(runner, ["--debug", "add", str(f), "--patient", p, "--specimen", s], expected_files=[f"MANIFEST/{f}.dvc"])
+
+    run(runner, ["--debug", "meta", "init"], expected_files=["META/DocumentReference.ndjson", "META/Patient.ndjson", "META/ResearchStudy.ndjson", "META/ResearchSubject.ndjson", "META/Specimen.ndjson"])
+    run(runner, ["--debug", "meta", "validate"])
+    run(runner, ["commit", "-m",  "add-specimen", "MANIFEST/", "META/", ".g3t", ".gitignore"])
+
+    run(runner, ["--debug", "push", "--dry-run"], expected_exit_code=0)
```

Comment:
Just to make sure, so this still satisfies (Nasim) or otherwise use case of adding additional valid FHIR data from an external source? 

I might have initially misunderstood, but I thought the use case was to add additional metadata that had a different project ID.

Reply:
The new "--skip-validation" is there to push data
https://github.com/ACED-IDP/gen3_util/blob/842e4faae802caec8470512f45fb98c81f8497ed/tests/unit/meta/test_meta.py#L58

Reply:
okay got it